docusaurus/docs/create-a-plugin/develop-a-plugin/manage-npm-dependencies.md

```diff
@@ -0,0 +1,58 @@
+---
+id: manage-npm-dependencies
+title: Manage NPM dependencies
+description: How to manage frontend NPM dependencies in Grafana plugins.
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - github
+  - npm
+  - dependencies
+sidebar_position: 5
+---
+
+# Manage frontend NPM dependencies in a Grafana plugin
+
+Frontend plugins in Grafana have their own unique dependencies, as well as dependencies that are shared with the Grafana application during runtime. This document focuses on how these shared dependencies, particularly the `@grafana` npm packages, are handled.
+
+It's important to understand that while a plugin specifies the expected versions of these dependencies in its `package.json` file, they are dynamically linked to the Grafana version at runtime.
+
+# Dynamic dependency linking
+
+When you develop your plugin, its `package.json` may reference a specific version of a `@grafana` npm package, such as `@grafana/ui: 9.5.1`.
+
+However, when the plugin is installed and executed within a Grafana instance, it inherits the version of the `@grafana` packages that the Grafana application is using. For example, if the Grafana version is 10.0.0, then the plugin uses version 10.0.0 of the shared `@grafana` dependencies from the Grafana application.
+
+# Dependency sharing mechanism
+
+To facilitate this dynamic dependency linking, Grafana employs SystemJS for loading frontend plugin code and sharing some of the Grafana application's npm dependencies with plugins.
```

Comment:
we could link to systemjs https://github.com/systemjs/systemjs

Reply:
done.

Reply:
I still can't see any links?

Reply:
i think the changes were made in the other file located in create-a-plugin/develop-a-plugin that I deleted as a duplicate. adding link.